### ISSUE TITLE
Wrong item conversion in array literals

### ISSUE THREAD

**Issue by InDIOS** [NectarJS/nectarjs#105]
Hi, thanks for include my nickname in the commit 🤣. I've tested this case:
```js
console.log(['ab', 'cd']);
```
It doesn't compile because there is a wrong conversion in this file `compiler/native/visitor/helpers/arrayExpression.js`,
```js
else _setter += `${_rnd}[${i}] = ${babel.generate(_path.elements[i]).code};`;
// -----------------------------------------------^ this cause that 'ab' compiles to a bad char in C++;
```
output:
```c++
// some code here
__cu3yea[0] = 'ab';
// ------------^ this is the problem, this is a int in C++
// other stuff
```
console output
```
 [ 24930,25444 ]
```
When you convert to an assigment, this case `'ab'` will output a wrong char in C++. You need to have a special case when a single quote string comes. Converting all javascript string cases (single quote, doble quotes and template literals) to a string literal in C++ and escape doble quotes whenever were necessary, could be a good solution to this issue.

PS: Sorry for my English

**Comment by adrien-thierry** [comment]
Hi @InDIOS ,

thank for the issue, all the single quoted strings literals should be change in double quoted in the babel StringLiteral visitor,

I will check what is happening, thx

**Comment by adrien-thierry** [comment]
Yes you're right, i just corrected it, i will push the update on npm

thx

**Comment by adrien-thierry** [comment]
Corrected in 540c7ccced214c893813b869735e1607527232b9

thx !
